### PR TITLE
Refactor/cleanup code

### PR DESCRIPTION
**Pull request by jasmyn2244**
___x_ Wrote Tests ____ Implemented __x__ Reviewed

## Neccesary checkmarks:

    [x] All Tests are Passing in all environments

    [x] The code will run locally

## Type of change

    [] New feature
    [] Bug Fix
    [x] Refactor

## Description of Change:

    description: cleaned up code, finished error tests
   

## Check the correct boxes

    [x] This broke nothing
    [] This broke some stuff
    [] This broke everything

## Testing Changes

    [] No Tests have been changed
    [x] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

    [x] My code has no unused/commented out code
    [x] My code has no binding.pry's
    [x] I have reviewed my code
    [x] I have commented my code, particularly in hard-to-understand areas
    [x] I have fully tested my code

### CODE REVIEW

File: cypress/integration/error-handling.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+describe('Error handling', () => {
+
+  it('Should show an error message with a 500 error', () => {
+    cy.intercept('POST', 'https://cryptic-garden-95478.herokuapp.com/graphql', (req) => {
```
Comment (krdurrett) [inline cypress/integration/error-handling.js:4]: This error handling testing looks great! Nice intercept

File: cypress/integration/filter-and-search-sightings-test.js [hunk 2]
```diff
@@ -7,21 +7,22 @@ describe('Filter the search by the cryptid', () => {
...
     cy.get('.sightings-button').click()
-    cy.get('input[type="text"]').type('C')
-    cy.get('input[type="text"]').type('o')
-    cy.get('input[type="text"]').type('l')
-    cy.get('input[type="text"]').type('o')
-    cy.get('input[type="text"]').type('r')
-    cy.get('input[type="text"]').type('a')
-    cy.get('input[type="text"]').type('d')
-    cy.get('input[type="text"]').type('o')
+      .get('input[type="text"]').type('Colorado')
```
Comment (krdurrett) [inline cypress/integration/filter-and-search-sightings-test.js:17]: Glad we could shorten this input test up

File: src/App.js [hunk 3]
```diff
@@ -9,20 +9,22 @@ import NavBar from './Components/NavBar';
...
+    <NavigationProvider>
+      <NavBar />
+      <ErrorModal />
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/singlecryptid/:name" element={<SingleCryptid />} />
+        <Route path="/allsightings/" element={<AllSightings />} />
+        <Route path="/singlesighting/:name/:id" element={<SingleSighting />} />
+        <Route path="/singlecryptidsightings/:name" element={<SingleCryptidSightings />} />
+        <Route path='/*' element={<Error404 />} />
```
Comment (krdurrett) [inline src/App.js:25]: Thanks for adding the Error404 container!

**Review by krdurrett — APPROVED** [review]
These are great changes to the testing

File: cypress/integration/filter-and-search-sightings-test.js [hunk 4]
```diff
@@ -7,21 +7,22 @@ describe('Filter the search by the cryptid', () => {
 
   it('Should filter the sightings by the alien cryptid', () => {
     cy.get('.sightings-button').click()
-    cy.get('.cryptid-button').click()
-    cy.get('#Alien').click()
-    cy.get('.logo-text').click()
+      .get('.cryptid-button').click()
```
Comment (GraceGardner) [inline cypress/integration/filter-and-search-sightings-test.js:10]: this looks a lot cleaner without the extra cy's

File: cypress/integration/filter-and-search-sightings-test.js [hunk 5]
```diff
@@ -7,21 +7,22 @@ describe('Filter the search by the cryptid', () => {
...
-    cy.get('input[type="text"]').type('o')
-    cy.get('input[type="text"]').type('l')
-    cy.get('input[type="text"]').type('o')
-    cy.get('input[type="text"]').type('r')
-    cy.get('input[type="text"]').type('a')
-    cy.get('input[type="text"]').type('d')
-    cy.get('input[type="text"]').type('o')
+      .get('input[type="text"]').type('Colorado')
+      .get('.all-sightings-container')
+      .get('.card-img').should('have.length', 3)
```
Comment (GraceGardner) [inline cypress/integration/filter-and-search-sightings-test.js:19]: Like how specific this is.

File: src/App.js [hunk 6]
```diff
@@ -9,20 +9,22 @@ import NavBar from './Components/NavBar';
...
-          <ErrorModal />
-          <Routes>
-            <Route path="/" element={<Home />} />
-            <Route path="/singlecryptid/:name" element={<SingleCryptid />} />
-            <Route path="/allsightings/" element={<AllSightings />} />
-            <Route path="/singlesighting/:name/:id" element={<SingleSighting />} />
-            <Route path="/singlecryptidsightings/:name" element={<SingleCryptidSightings />} />
-          </Routes>
-      </NavigationProvider>
+    <NavigationProvider>
```
Comment (GraceGardner) [inline src/App.js:16]: Like that sad path testing here

**Review by GraceGardner — APPROVED** [review]
Nicely done! Looks great!

**Pull request merged**